### PR TITLE
Added check to avoid keyring call to get server password if save_password flag is not set

### PR DESCRIPTION
**Pull request by nikhil-mohite**

### CODE REVIEW

File: web/pgadmin/browser/__init__.py [hunk 1]
```diff
@@ -724,18 +724,19 @@ def set_master_password():
 
         try:
             all_server = Server.query.all()
+            saved_password_servers = [server for server in all_server if
+                                      server.save_password]
             # pgAdmin will use the OS password manager to store the server
             # password, here migrating the existing saved server password to
             # OS password manager
-            if keyring.get_password(
+            if len(saved_password_servers) > 0 and keyring.get_password(
```
Comment (adityatoshniwal) [inline web/pgadmin/browser/__init__.py:732]: so even if one server has saved password then it will call get password for all?

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/browser/__init__.py (hunk 1 above)
Comment (nikhil-mohite) [reply web/pgadmin/browser/__init__.py:732]: No for all, it is for specific server only.

**Comment by adityatoshniwal** [comment]
@nikhil-mohite  all the python tests are failing for this.

**Comment by adityatoshniwal** [comment]
@nikhil-mohite  The tests are still failing for ubuntu.

**Pull request merged**